### PR TITLE
Document binary artifacts

### PR DESCRIPTION
**Pull request by jackkleeman**

### CODE REVIEW

File: docs/restate/deployment.md [hunk 1]
```diff
@@ -4,9 +4,11 @@ description: "Deploy Restate on Kubernetes with this guide."
 ---
 
 # Deployment
+Restate is currently a single binary that contains everything you need. You can obtain the binary from the
+[releases page](https://github.com/restatedev/restate/releases), as part of a
+[Docker image](https://hub.docker.com/r/restatedev/restate), or with `npm install @restatedev/restate-server`.
```
Comment (tillrohrmann) [inline docs/restate/deployment.md:9]: What about the homebrew?

**Review by tillrohrmann — COMMENTED** [review]
Thanks for creating this PR @jackkleeman. I am wondering whether we shouldn't document where to get Restate from a bit more prominently than only on the deployment page. E.g. one place that is a bit more prominent could be the quickstart. I guess we should then also update the Readme's of the relevant repos to list the different options or link to the documentation.

File: docs/restate/deployment.md (hunk 1 above)
Comment (jackkleeman) [reply docs/restate/deployment.md:9]: good point

**Comment by jackkleeman** [comment]
@tillrohrmann i didnt want to add to quickstart because the commands there are all written for docker and this has the advantage of using ephemeral storage. do we want to change them all to use local binary?

**Comment by tillrohrmann** [comment]
Maybe we just say that these are the different ways to obtain Restate and then for the rest of the quickstarts we use the docker command?

> @tillrohrmann i didnt want to add to quickstart because the commands there are all written for docker and this has the advantage of using ephemeral storage. do we want to change them all to use local binary?

Maybe we just say that these are the different ways to obtain Restate and then for the rest of the quickstarts we use the docker command?

**Review by tillrohrmann — APPROVED** [review]
LGTM. +1 for merging.

**Pull request merged**